docs/experimental/lakectl_local_posix.md

```diff
@@ -0,0 +1,99 @@
+* Support user and group preservation for files and folders managed via lakectl **local for POSIX compliant filesystems only**
+
+
+{: .note}
+> This feature is currently supported for Unix based Operating Systems only
+
+{: .warning }
+> Please make sure to contact the lakeFS team before enabling any experimental features!
+
+## Configuration 
+
+By default, this feature is disabled. Enabling file permission tracking is done via the configuration variable 
+`experimental.local.posix_permissions_enabled` which can be added to the lakectl.yaml file:
+
+   ```yaml
+   credentials:
+     access_key_id: AKIAIOSFDNN7EXAMPLEQ
+     secret_access_key: wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+   metastore:
+     glue:
+       catalog_id: ""
+     hive:
+       db_location_uri: file:/user/hive/warehouse/
+       uri: ""
+   server:
+     endpoint_url: http://localhost:8000/api/v1
+   experimental:
+     local:
+       posix_permissions:
+         enabled: true
+   ```
+
+It is also possible to enable this feature via the corresponding environment variable:
+
+```
+LAKECTL_EXPERIMENTAL_LOCAL_UNIX_PERMISSIONS_ENABLED
+```
+
+## Usage
+
+lakectl local will automatically track changes in file permissions and ownership as part of the sync with the remote server.
+During the first sync with a remote path, lakectl local will need to update all the objects under this path in order to sync
+the permissions with lakeFS.
+When first creating the file and directory structure from an existing remote path the default permissions are set:
+- `0o0666-umask` mode for files
+- `0o0777-umask` mode for directories
+- GID and UID of the current user
+Please note that once a local path is synchronized with a remote path with permissions tracking enabled, you must continue to work with this remote path
+with the feature enabled.
+
+### Changes from default behavior
+
+In order to support this feature, some changes needed to be made in lakectl local, and as such the behavior when the feature is enabled is a bit different from the default behavior of lakectl local.
+The main differences are:
+1. To support tracking of directory permissions and ownership, lakectl local will now write directory markers to the remote server, which will hold this information on the remote object.
+These markers are zero sized objects with paths ending in "/" designed to indicate a directory in S3 and used to save the directory information.
+2. Syncing between local and remote paths will take into account changes in file ownership and permissions. In case of change in one of these, the file will show as modified
+
+## Limitations / Warnings
+
+- All machines on which lakectl local is used with POSIX permissions must share UIDs and GIDs.
+- Tracked modes will only be accessible via lakectl local and not through the UI, or any lakeFS API calls (e.g., statObject) or S3 gateway operations (e.g., getObject).
```

Comment:
If these are user object metadata then AFAIK these are exposed in statObject and the S3 gateway.